packages/terra-filter-view/src/_Frame.module.scss

```diff
@@ -0,0 +1,181 @@
+@import '~terra-mixins';
+
+:local {
+  .select-parent {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+  }
+
+  .select {
+    background-color: var(--terra-filter-select-background-color, #fff);
+    color: var(--terra-filter-select-color, #1c1f21);
+    display: inline-flex;
+    font-size: var(--terra-filter-select-font-size, 1.143rem);
+    line-height: var(--terra-filter-select-line-height, 1.5);
+    position: relative;
+  }
+
+  .display {
+    overflow: hidden; // IE 10
+  }
+
+  .button {
+    border: var(--terra-filter-search-button-border, 0.071em solid #dedfe0);
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0.25em;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0.25em;
+    display: inline-block;
+    flex: 0 0 auto;
+    margin: 0;
+
+    &:hover {
+      background-color: var(--terra-filter-search-button-hover-background-color, #b9bbbc);
+      border-color: var(--terrra-filter-search-button-hover-border-color, #c8cacb);
+    }
+
+    &:active {
+      background-color: var(--terra-filter-search-button-active-background-color, #6f7477);
+      border-color: var(--terra-filter-search-button-active-border-color, #52585b);
+      color: #fff;
+    }
+  }
+
+  .clear-button {
+    color: var(--terra-filter-clear-button-color, #b9bbbc);
+    margin-left: 0.3571428571rem;
+    margin-right: 0.3571428571rem;
+    position: absolute;
+    right: 30px;
+    top: 7px;
```

Comment:
If the filter view is themed to be larger or smaller I think these static values may position the button inaccurately.

Flexbox positioning the clear button and the input together may position content a little better.